src/components/ReminderCtaButton.tsx

```diff
@@ -0,0 +1,186 @@
+import React, { useState } from 'react';
+import { css, ThemeProvider } from '@emotion/react';
+import { neutral, brandAlt, news, body, space } from '@guardian/source-foundations';
+import { Button } from '@guardian/source-react-components';
+
+import {
+    buildReminderFields,
+    createReminder,
+    ReminderStage,
+    ReminderStatus,
+} from '../logic/reminders';
+
+import { FetchEmail } from '../types/dcrTypes';
+import { TrackClick } from '../utils/tracking';
+import { LoadingDots } from './CtaLoadingDotsAnimation';
+
+// Custom theme for Button/LinkButton
+// See also `tertiaryButtonOverrides` below.
+const buttonStyles = {
+    textPrimary: neutral[7],
+    backgroundPrimary: brandAlt[400],
+    backgroundPrimaryHover: brandAlt[300],
+    textSecondary: neutral[7],
+    backgroundSecondary: neutral[93],
+    backgroundSecondaryHover: neutral[86],
+    borderSecondary: neutral[86],
+};
+
+const contributionsTheme = {
+    button: buttonStyles,
+    link: buttonStyles,
+};
+
+const styles = {
+    buttonWrapperStyles: css`
+        margin: ${space[4]}px ${space[2]}px ${space[1]}px 0;
+        display: flex;
+        flex-direction: column;
+        flex-wrap: wrap;
+        justify-content: flex-start;
+        align-items: flex-start;
+        max-width: 50%;
+    `,
+    buttonMargins: css`
+        margin: ${space[1]}px ${space[2]}px ${space[1]}px 0;
+    `,
+    button: css`
+        background-color: ${news[400]};
+        color: ${neutral[100]};
+        &:hover {
+            background-color: ${news[400]};
+        }
+    `,
+    thankYouText: css`
+        ${body.medium({ fontWeight: 'bold' })};
+        color: ${neutral[0]};
+        margin-top: 12px;
+    `,
+    errorText: css`
+        color: ${neutral[0]};
+        margin-top: 12px;
+    `,
+    remindMeButtonOverrides: css`
+        background-color: transparent !important;
+        color: ${neutral[7]} !important;
+
+        :hover {
+            background-color: ${neutral[86]} !important;
+        }
+    `,
+    smallPrint: css`
+        margin-top: 8px;
+    `,
+};
+
+interface RemindMeButtonProps {
+    disabled: boolean;
+    ctaText: string;
+    onClick: () => void;
+}
+
+const RemindMeButton = ({ disabled, ctaText, onClick }: RemindMeButtonProps) => (
+    <div css={styles.buttonMargins}>
+        <ThemeProvider theme={contributionsTheme}>
+            <Button
+                disabled={disabled}
+                onClick={() => onClick()}
+                priority="tertiary"
+                css={styles.remindMeButtonOverrides}
+            >
+                {ctaText}
+            </Button>
+        </ThemeProvider>
+    </div>
+);
+
+interface SmallPrintProps {
+    month: string;
+}
+
+const SmallPrint = ({ month }: SmallPrintProps) => (
+    <div css={styles.smallPrint}>
```

Comment:
Has there been design input on this bit? In the screenshots the font doesn't look very Guardian-y!

Reply:
No design input. I pasted a screenshot into the TX chat a while back o get feedback. Will look at making sure the correct font is being used here

Reply:
Fonts fixed. Images updated in the PR description